commands/config.go

```diff
@@ -132,9 +155,33 @@ func setValue(path []string, obj reflect.Value, newValue interface{}) error {
 				return errors.New("Target value type is not supported")
 			}
 		}
-	} else {
-		return errors.New("Path object type is not supported")
+	} else if len(path) > 1 {
```

Comment:
no `else` needed. I'd get rid of this `if` altogether, like: 
`if len(path)==0{
    return errors.New("Path object type is not supported")
}`